<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="centralwidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="figureLabel">
         <property name="text">
          <string>Фигура:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="figureComboBox">
         <item>
          <property name="text">
           <string>Конь</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ладья</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ферзь</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="startPositionLabel">
         <property name="text">
          <string>Старт:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="startPositionLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="endPositionLabel">
         <property name="text">
          <string>Финиш:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="endPositionLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="calculateButton">
         <property name="text">
          <string>Рассчитать</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="resultTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="chessBoardTableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>421</width>
      <height>431</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>8</number>
    </property>
    <property name="columnCount">
     <number>8</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
